src/smif/decision/decision.py

```diff
@@ -26,13 +29,13 @@ class DecisionManager(object):
     form of a generator which allows the model runner to iterate over the collection of
     independent simulations required at each step.
 
-    A DecisionManager collates the output of the decision algorithms and
+    The DecisionManager collates the output of the decision algorithms and
     writes the post-decision state through a DataHandle. This allows Models to access a given
     decision state (identified uniquely by timestep and decision iteration id).
 
-    A DecisionManager passes a DataHandle down to a
-    DecisionModule, allowing the DecisionModule to access model results from previous timesteps
-    and decision iterations when making decisions.
+    The DecisionManager passes a DataHandle down to a DecisionModule,
+    allowing the DecisionModule to access model results from previous timesteps
+    and decision iterations when making decisions
```

Comment:
Also try and keep the docstrings updated here

Reply:
Fixed